media-libs/libjpeg-turbo/libjpeg_turbo-1.5.3.recipe

```diff
@@ -0,0 +1,149 @@
+SUMMARY="MMX, SSE, and SSE2 SIMD accelerated JPEG library"
+DESCRIPTION="libjpeg-turbo is a JPEG image codec that uses SIMD instructions \
+(MMX, SSE2, AVX2, NEON, AltiVec) to accelerate baseline JPEG compression and \
+decompression on x86, x86-64, ARM, and PowerPC systems. On such systems, \
+libjpeg-turbo is generally 2-6x as fast as libjpeg, all else being equal. On \
+other types of systems, libjpeg-turbo can still outperform libjpeg by a \
+significant amount, by virtue of its highly-optimized Huffman coding routines.
+In many cases, the performance of libjpeg-turbo rivals that of proprietary \
+high-speed JPEG codecs.
+libjpeg-turbo implements both the traditional libjpeg API as well as the less \
+powerful but more straightforward TurboJPEG API. libjpeg-turbo also features \
+colorspace extensions that allow it to compress from/decompress to 32-bit and \
+big-endian pixel buffers (RGBX, XBGR, etc.), as well as a full-featured Java \
+interface.
+libjpeg-turbo was originally based on libjpeg/SIMD, an MMX-accelerated \
+derivative of libjpeg v6b developed by Miyasaka Masaru. The TigerVNC and \
+VirtualGL projects made numerous enhancements to the codec in 2009, and in \
+early 2010, libjpeg-turbo spun off into an independent project, with the goal \
+of making high-speed JPEG compression/decompression technology available to \
+a broader range of users and developers."
+HOMEPAGE="https://www.libjpeg-turbo.org/"
+COPYRIGHT="1991-2018 Thomas G. Lane, Guido Vollbeding
+	1997-2010 Guido Vollbeding
+	2000-2017 D. R. Commander
+	2009 Bill Allombert
+	1999-2006 MIYASAKA Masaru
+	2011 Siarhei Siamashka
+	2013 Linaro Limited
+	2014 MIPS Technologies, Inc.
+	2015 Google, Inc.
+	2015 Matthieu Darbois"
+LICENSE="BSD (3-clause)
+	IJG
+	Zlib"
+REVISION="1"
+SOURCE_URI="https://github.com/libjpeg-turbo/libjpeg-turbo/archive/$portVersion.tar.gz"
```

Comment:
https://www.libjpeg-turbo.org/ has a link called "Official Binaries and Source Tarballs" which points to SourceForge.
Gentoo also uses the tarball from SF. So maybe we should use them too.
```diff
-SOURCE_URI="https://github.com/libjpeg-turbo/libjpeg-turbo/archive/$portVersion.tar.gz"
+SOURCE_URI="https://downloads.sourceforge.net/libjpeg-turbo/libjpeg-turbo-$portVersion.tar.gz"
-CHECKSUM_SHA256="1a17020f859cb12711175a67eab5c71fc1904e04b587046218e36106e07eabde" 
+CHECKSUM_SHA256="b24890e2bb46e12e72a79f7e965f409f4e16466d00e1dd15d93d73ee6b592523"
```
BTW, if you switch to the official tarball from SF we won't need the `SOURCE_FILENAME` anymore :)

Remark: I did not test yet if both archives are equivalent. I guess they are, but I need to kick a build.